### PR TITLE
Add grade, and add activity view

### PR DESCRIPTION
**Pull request by wyder110**
This is not polished, but I think its good enough for the first presentation.

### CODE REVIEW

**Review by marcinz99 — APPROVED** [review]
Looks fine.

File: src/main/resources/templates/activity_lecturer.html [hunk 1]
```diff
@@ -16,11 +16,38 @@ <h2 class="pb-2" th:text="${activity.getName()}">Aktywność</h2>
...
+    </div>
+    </section>
+
     <p class="pb-3" style="border-bottom: 3px solid #343a40;">
         Tutaj możesz podejrzeć wszystkie oceny swoich podopiecznych w ramach tej aktywności.
     </p>
+
+
+
+    <table class="table table-hover table-bordered bg-light">
```
Comment (anosek-an) [inline src/main/resources/templates/activity_lecturer.html:35]: I think we should also display description of the Grade - right now there's no way of distinguishing grades of the same user. Descriptions are empty, but we can add them in DataLoader.

File: src/main/resources/templates/new_grade.html [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    <div class="form-group">
+      <label for="userName">Wartość oceny:</label>
+      <select class="form-control" id="userName" th:name="userName">
+        <option th:each="student : ${course.getEnrolledStudents()}" th:value="${student.getEmail()}" th:text="${student.getEmail()}"></option>
+      </select>
+    </div>
+    <div class="form-group">
+      <label for="gradeValue">Wartość oceny:</label>
+      <input type="number" class="form-control" id="gradeValue" th:name="gradeValue" placeholder="Wartość oceny">
+    </div>
```
Comment (anosek-an) [inline src/main/resources/templates/new_grade.html:37]: Also, here we should be able to add grade description.

**Review by anosek-an — APPROVED** [review]
Some changes need to be done but great job in such short amount of time. 
Also, we need to discuss the grade model - it is possible that grades in one activities should be standardized between students.

**Pull request merged**